<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pause</class>
 <widget class="QDialog" name="Pause">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>暂停</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>21</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>保存游戏</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="loadButton">
      <property name="text">
       <string>载入游戏</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="openmenuButton">
      <property name="text">
       <string>返回主界面</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="continueButton">
      <property name="text">
       <string>继续游戏</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
